src/collective/cover/browser/compose.py

```diff
@@ -264,11 +162,13 @@ def render(self):
         tile_id = self.request.form.get('tile-id')
         uuid = self.request.form.get('uuid')
         if tile_type and tile_id and uuid:
-            tile = self.context.restrictedTraverse('{0}/{1}'.format(tile_type, tile_id))
+            tile = self.context.restrictedTraverse(
+                '{0}/{1}'.format(tile_type, tile_id))
             if IListTile.providedBy(tile):
                 tile.remove_item(uuid)
                 # reinstantiate the tile to update its content on AJAX calls
-                tile = self.context.restrictedTraverse('{0}/{1}'.format(tile_type, tile_id))
+                tile = self.context.restrictedTraverse(
+                    '{0}/{1}'.format(tile_type, tile_id))
```

Comment:
@rodfersou wouldn't be better to do `'{0}/{1}'.format(tile_type, tile_id)` only once?